Monika After Story/game/script-topics.rpy

```diff
@@ -4790,6 +4790,163 @@ label monika_urgent:
+                m 1e "Perhaps I should listen to more of it, you dig?"
+                m 3l "Ehehe~"
+                m 3e "I'm sorry. I find the slang curious, too."
+                m 4b "If you want to mute the music here~"
+                m "And put on some jazz~"
+                m 3j "I'll happily listen along."
+                m 3k "Let's break it down, baby!"
+                m 5a "Ehehe~"
+                m 2b "I hope you don't mind putting on orchestral now and then though, too."
+            
+            "Rap.":
+                m 1d "You like rap?"
+                m 1q "I'll be honest with you right off the bat..."
+                m 1i "I used to hate rap music."
+                m 2d "Used to~"
+                m 1a "Until I realized what an incredible form of literature it is."
+                m 1d "Having to match your words to rhyme and rythm~"
+                m 1a "That's certainly something to respect."
+                m 1n "When you aren't using it, I have used your web browser~"
+                m "And came across some rather nice songs~"
+                m 1a "So if you ever want to listen to rap~"
+                m 2k "Have at it, my love!"
+                m 2e "Just be sure to mute this music, so that way I can hear yours."
+                m 1e "And if I could make a request~"
+                m 5a "The occasional orchestra piece would be amazing~"
+                m 1b "But it's up to you."
+            
+            "Pop.":
+                m 1d "Pop music?"
+                m 2a "That's certainly a popular genre."
+                m 3j "Ehehe~"
+                m 3c "I will admit, though~"
+                m 3o "A lot of it still sounds the same to me."
+                m 3m "Maybe it's because there's so much of it on the radio~"
+                m 3a "But I'm sure you have your own playlist of the very best of pop!"
+                m 4b "If you ever want to play it~"
+                m "Mute the music here so I can listen, too."
+                m 5a "And don't forget to add an orchestral piece or two!"
+            
+            "Folk."
+                m 2b "The oldest of the old, huh?"
+                m 2n "Wow, folk music sure has been around a long time."
+                m 3a "I guess it sort of etched itself into history, though."
+                m 1c "A lot of folk music doesn't even have a known composer."
+                m 1d "I find that odd myself."
+                m 2g "Why wouldn't you want credit for something you worked so hard on?"
+                m 3n "But maybe it isn't that simple."
+                m 4a "In any case, be sure to play some folk, okay?"
+                m 3c "But mute the music here, otherwise I can't listen along."
+                m 3e "And..."
+                m "If you could play the occasional bit of orchestra~"
+                m 5a "That's be amazing too, my love."
+            
+            "It's not here...":
```

Comment:
Shouldn't it be "It's not _in_ here..."?
Don't take my word for it tho.

Reply:
Yeah, I'll tweak that.